### ISSUE TITLE
About the accuracy computation.

### ISSUE THREAD

**Issue by rookiecm** [Zasder3/train-CLIP#6]
Thanks for sharing you code. I am a little bit confused about the accuracy computation in L69-70 in wrapper.py:

`acc_i = (torch.argmax(image_logits) == ground_truth).sum()`
`acc_t = (torch.argmax(image_logits.t()) == ground_truth).sum()`

It seems that torch.argmax retures the max value index accross all dimensions while _ground_truth_ is with each row or column. Should we change to?

`acc_i = (torch.argmax(image_logits, 0) == ground_truth).sum()`
`acc_t = (torch.argmax(image_logits.t(), 0) == ground_truth).sum()`.

Thanks.

**Comment by Zasder3** [comment]
It looks like it! Thanks for catching this bug. I'll send a fix right now.

**Comment by Zasder3** [comment]
Made an update and also condensed it into a single term due to the diagonal being the same. Should look good now!
